docs/deployment.md

```diff
@@ -18,18 +18,18 @@ Inhalt:
 
 # {{ page.title }}
 {: .no_toc }
-So far, we used to run, debug and test plug-ins by loading them in *PiWeb Auto Importer* directly from their build output directory via command line parameter <span class="nowrap">`-pluginSearchPaths`</span>. While easy for testing and debugging, this is, of course, not a practicable way of distributing a plug-in after its release and even less so to deploy it in a production environment. In this article we will show you how to create an easily distributable plug-in package and how to install it in a *PiWeb Auto Importer* not running in development mode.
+So far, we used to run, debug and test plug-ins by loading them in *PiWeb Auto Importer* directly from their build output directory via command line parameter <span class="nowrap">`-pluginSearchPaths`</span>. While easy for testing and debugging, this is, of course, not a robust way to deploy a plug-in in a production environment. In this article we will show you how to create an easily distributable plug-in package and how to install it in a *PiWeb Auto Importer* not running in development mode.
 
 ## Table of Contents
 {: .no_toc }
 1. TOC
 {:toc}
 
 ## Creating a plug-in package
-A *PiWeb Installable Plug-in* (.pip) package file is automatically created on every build of your plug-in as long as the *Import SDK* nuget is referenced by the project. The file is written to `$(PackageOutputPath)` which usually is (depending on your build configuration) either `bin\Debug` or `bin\Release`. A .pip file contains a the complete plug-in implementation and does not have any other dependencies. As a single file, it is easy to distribute.
+A *PiWeb Installable Plug-in* (.pip) package file is automatically created on every build of your plug-in project as long as the *Import SDK* nuget is referenced by the project. The created .pip file is written to `$(PackageOutputPath)` which usually is either `bin\Debug` or `bin\Release`, depending on your build configuration. A .pip file contains the complete plug-in implementation and does not have any other dependencies. As it is a single file, it is easy to distribute.
 
 {: .note}
->You may switch building the package on and off by setting the `GeneratePluginPackageOnBuild` property in the project file:
+>You can switch the building of the package on or off by setting the `GeneratePluginPackageOnBuild` property in the project file:

```

Comment:
Maybe this way:
"You can switch the building of the package on or off by setting the `GeneratePluginPackageOnBuild` property in the project file by setting it to true or false:"

We should also mention what's used as default value if the setting is not defined in the project file.

Reply:
If the settings is not present, no package file will be created. However, the setting property should initially be there because of the project template, so I think it is enough to say what it does.

Reply:
I reworded the section somewhat to avoid the true/false and on/off confusion.